docs/requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy
+ghp-import
```

Comment:
In the past we used a [github action](https://github.com/brainets/frites/blob/master/.github/workflows/test_doc.yml#L57) to push the doc to the gh-pages branch. Here you use a python package instead. What are the advantages of this choice?

Reply:
The tutorial I referred to had used `ghp-import` and mentioned that it would not give compatibility issues if the github-pages-deploy-action version changes

Reply:
True but you can configure a bot performing weekly updates. That way it's automatic and always up-to-date. See [dependabot](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot) and here's an example of the [configuration](https://github.com/brainets/frites/blob/master/.github/dependabot.yml).